src/main/java/org/littil/api/teacher/service/TeacherService.java

```diff
@@ -0,0 +1,74 @@
+package org.littil.api.teacher.service;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.littil.api.exception.ServiceException;
+import org.littil.api.teacher.repository.TeacherEntity;
+import org.littil.api.teacher.repository.TeacherRepository;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.persistence.PersistenceException;
+import javax.transaction.Transactional;
+import javax.validation.Valid;
```

Comment:
deze uiteindelijk vast naar jakarta ee api

Reply:
Created Task for it.